src/sentry/api/helpers/group_index/update.py

```diff
@@ -1,7 +1,7 @@
 from __future__ import annotations
 
 from collections import defaultdict
-from typing import Any, Mapping, MutableMapping, Sequence
+from typing import Any, Dict, Mapping, MutableMapping, Sequence
```

Comment:
Supposedly now we're supposed to use `dict`.

https://typing.readthedocs.io/en/latest/source/best_practices.html#built-in-generics